src/main/webapp/staticFeeds/sample_notifications.json

```diff
@@ -24,5 +24,19 @@
         "actionAlt" : "Activate Services",
         "dismissable" : true
       }
-    ]
+    ],
+  "dismissedNotifications" : [
```

Comment:
Valid ideas @apetro . Had similar thoughts. I like the idea. Note that this does complicate calculating the number of new notifications. That would be my only concern, as we would have to do date compares to see if its new.